### PR TITLE
Add config for stardust-testnet

### PR DESCRIPTION
**Pull request by muXxer**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -213,27 +163,14 @@
     "advancementRange": 150
   },
   "spammer": {
-    "message": "IOTA - A new dawn",
+    "message": "We are all made of stardust.",
     "index": "HORNET Spammer",
```
Comment (alexsporn) [inline config.json]: this should be tag and tagSemiLazy

File: docker-compose.yml [hunk 2]
```diff
@@ -24,10 +24,6 @@ services:
...
     volumes:
       - ./config.json:/app/config.json:ro
-      - ./config_devnet.json:/app/config_devnet.json:ro
-      - ./config_comnet.json:/app/config_comnet.json:ro
       - ./peering.json:/app/peering.json
       - ./profiles.json:/app/profiles.json:ro
-      - ./mainnetdb:/app/mainnetdb
-      - ./p2pstore:/app/p2pstore
-      - ./snapshots/mainnet:/app/snapshots/mainnet
+      - ./stardust_testnet:/app/stardust_testnet
```
Comment (alexsporn) [inline docker-compose.yml:32]: I actually don't like that the network name is hardcoded inside the docker container

File: config.json (hunk 1 above)
Comment (muXxer) [reply config.json]: Fixed

File: docker-compose.yml (hunk 2 above)
Comment (muXxer) [reply docker-compose.yml:32]: I would leave it as it is for now for the alpha tests.

**Review by shufps — APPROVED** [review]
looks good to me :thinking:

**Pull request merged**